libpod/runtime_ctr.go

```diff
@@ -475,6 +475,14 @@ func (r *Runtime) setupContainer(ctx context.Context, ctr *Container) (_ *Contai
 		if isAnonymous {
 			volOptions = append(volOptions, withSetAnon())
 		}
+		if len(vol.DriverOpts) > 0 {
+			parsedOptions, err := VolumeOptions(vol.DriverOpts)
+			if err != nil {
+				return nil, err
+			}
+			volOptions = append(volOptions, parsedOptions...)
+		}
+		//volOptions.Options = vol.DriverOpts
```

Comment:
if we don't need this code, let's drop it

Reply:
fixed